Dockerfile

```diff
@@ -3,6 +3,7 @@ FROM docker.io/debian:11-slim
 ARG VCS_VER
 ARG VCS_REF
 ARG DEBIAN_FRONTEND=noninteractive
+ARG DEBCONF_NONINTERACTIVE_SEEN=true
```

Comment:
So this ARG is not related to the TZ change (this PR) right? 
Otherwise it is not needed as ARG as TZ config is only done during runtime.

If it is useful for other tasks during build time we can keep it, otherwise i would remove it again

Reply:
Found it [here](https://wiki.debian.org/Multistrap/Environment), not 100% sure though whether we absolutely need it. I guess for the current build, we don't...

Reply:
Not needed, don't keep it? 🤷

Reply:
Ahhh, yes... :D

Reply:
Done in 4d261e2ec2eb841125be99c0cf14ad30edbca22c